README.md

```diff
@@ -19,23 +19,14 @@
 - [Steps to add yourself to the contributers list](https://github.com/alisolanki/Welcome-to-Open-Source/blob/master/CONTRIBUTING.md)
 
 <p align="center">
-    <a href="https://www.twitter.com/alisolankii">
-        <img height="50" src="https://cdn-icons-png.flaticon.com/512/4096/4096132.png"/>
+    <a href="https://www.linkedin.com/in/vikasgaur1709/">
+        <img height="50" src="https://media.licdn.com/dms/image/D4D03AQH3tTh5x3JBOQ/profile-displayphoto-shrink_200_200/0/1666032334943?e=1724889600&v=beta&t=vh34w31VdiVfcHkhLSaofEBLeGmu0PFwQXG7l6Aspgo"/>
     </a>
-    <a href="https://linkedin.com/in/alisolanki">
-        <img height="50" src="https://user-images.githubusercontent.com/46517096/166973395-19676cd8-f8ec-4abf-83ff-da8243505b82.png"/>
-    </a>
-    <a href="https://www.youtube.com/AliSolanki">
-        <img height="50" src="https://github.com/Hsrah00/icons/blob/main/6214533_logo_youtube_icon.png"/>
-    </a>
-    <a href="https://www.instagram.com/alisolankii">
-        <img height="50" src="https://user-images.githubusercontent.com/46517096/166974368-9798f39f-1f46-499c-b14e-81f0a3f83a06.png"/>
+    <a href="https://github.com/Vikas-Gaur-1">
+        <img height="50" src="https://avatars.githubusercontent.com/u/66732930?v=4"/>
     </a>
 </p>
-  
-<p align="center">
-  <img src= "https://media.giphy.com/media/3xz2Bw12fe9iyG06v6/giphy.gif">
-</p>
+
```

Comment:
These changes are not necessary at all. Please revert them in order to get this PR merged.

Reply:
I have done the necessary changes of reverting in the README file.